src/qt/overviewpage.cpp

```diff
@@ -550,6 +550,7 @@ void OverviewPage::showEvent(QShowEvent *event){
     if (fHide != filter->orphansHidden())
         hideOrphans(fHide);
 
+    /*
     QGraphicsOpacityEffect *eff = new QGraphicsOpacityEffect(this);
```

Comment:
This and several other (animation?) effects have been remarked out in this PR. Was this a debugging change, or is there a reason to remove these?

Reply:
If i remember correctly, this was because the animation is clashing with the styling (causes flicker or just looks bad and is unnecessary). If you add it back in, would just need an eyeball check as it is all cosmetic changes.